docs/src/dev-guide/contributing-linting.md

```diff
@@ -0,0 +1,20 @@
+# Linting
+
+Before submitting a PR, ensure you've run our linting tools and either fixed any violations or
+suppressed the warning. To run our linting workflows locally, you'll need [Task][1]. Alternatively,
```

Comment:
For 1 and 2 here can we use plain english names instead like we've switched to in other parts of the docs? E.g. [Task][task] etc.